mixin-styles-gb/inc/editor-palette.php

```diff
@@ -0,0 +1,126 @@
+<?php
```

Comment:
These are not needed anymore, you can add both the colors and gradients to experimental-theme.json